lib/org/chromium/webidl/WebKitParser.js

```diff
@@ -52,10 +55,23 @@ foam.CLASS({
                   sym('ExtendedAttributeNamedArgList'),
                   sym('ExtendedAttributeIdentifierOrString'),
                   sym('ExtendedAttributeArgList'),
+                  sym('ExtendedAttributeIdentifier'),
```

Comment:
This is actually not required anymore either. This was necessary because files within `win/interfaces` had attributes such as `UUID(78f416b8-e30d-4886-9884-6ace2f7e21f7)`